src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -127,7 +118,7 @@ public function createAction(Request $request): Response
      *
      * @return Response
      */
-    public function editAction($featureId, Request $request): Response
+    public function editAction(int $featureId, Request $request): Response
```

Comment:
Yep it's a better fix than force-casting it later, so now you can clean `(int) $featureId` to `$featureId`